src/runtime/native/runtime.mli

```diff
@@ -41,16 +45,18 @@
 val register_file :
   bisect_file:string option ->
   bisect_silent:string option ->
+  bisect_sigterm:bool ->
   filename:string ->
   points:int list ->
     [`Visit of (int -> unit)]
-(** [register_file ~bisect_file ~bisect_silent file ~point_count
+(** [register_file ~bisect_file ~bisect_silent ~bisect_sigterm file ~point_count
     ~point_definitions] indicates that the file [file] is part of the
     application that has been instrumented. [point_definitions] is a serialized
     [Common.point_definition list] giving the locations of all points in the
     file. The returned callback is used to increment visitation counts.
-    [bisect_file] (resp. [bisect_silent]) is a default value for the environment
-    variable [BISECT_FILE] (resp. [BISECT_SILENT]). *)
+    [bisect_file] (resp. [bisect_silent] and [bisect_sigterm ) is a default
```

Comment:
The markup `[bisect_sigterm )` looks wrong here.

Reply:
Fixed in 346c9d2.